Utilities/PUID.cc

```diff
@@ -1,9 +1,10 @@
 #include <vector>
 #include <string>
+#include <iostream>
 
 //working points for PU jet ID implemented as in https://twiki.cern.ch/twiki/bin/view/CMS/PileupJetIDUL
 std::vector<bool> PUID(float pT_, float absEta_, float PUIDdisc_, string year_){
- std::vector<bool> output;
+ std::vector<bool> output = {true, true, true};
```

Comment:
I found one event which caused a seg fault, because no condition in here was met, somehow. So I made sure to also give this an initialization...

Reply:
Shouldn't it be false false false by default? That's what I changed to.

Reply:
It's fed into PUIDpasses, so true means here that the jet is viable. In case this file, which has all the arguments whether a jet should fail, has no output, it should be viable by default instead. E.g. the PUID does not cover any jet with a pT above 50 GeV. Therefore, everything above 50 GeV should pass automatically.